### PR TITLE
Edit README.md file and a few code comments

### PR DESCRIPTION
**Pull request by Muhammad-Altabba**
After doing an [overall review](https://github.com/web3/web3.js/issues/5588), the code looks good, and the other open issues seem to identify all the concerns. 

However, I found some needed edits to README.md file and a few code comments that have been added to this MR.

### CODE REVIEW

File: README.md
Comment (mpetrunic) [inline README.md:21]:
```suggestion
npm install @chainsafe/web3.js-chainlink-plugin
```

File: README.md
Comment (mpetrunic) [inline README.md:27]:
```suggestion
yarn add @chainsafe/web3.js-chainlink-plugin
```

File: README.md
Comment (mpetrunic) [inline README.md:33]: any reason why the example is removed?

File: README.md
Comment (jdevcs) [inline README.md:37]:
```suggestion
You may like to run the tests and examine their code to see how to use the plugin. And this would be also useful if you are a plugin writer.
```

**Review by jdevcs — COMMENTED** [review]
Just a tiny change required

File: README.md
Comment (Muhammad-Altabba) [reply README.md:33]: Sorry, this seems to be un-updated and not a complete part of the code of the written plugin and not an example of  using it: https://github.dev/ChainSafe/web3.js-plugin-chainlink/blob/620ec464385f9eee0eae43d1e37a41db0ddc7cf7/src/index.ts#L10

**Review by jdevcs — APPROVED** [review]

File: README.md
Comment (mpetrunic) [reply README.md:33]: It would be better to update the example than to explain how to use it, no?

Comment (Muhammad-Altabba) [reply README.md:33]: That is right, we will add an article on how to use this plugin and it would contain examples for js and ts. This would be part of https://github.com/web3/web3.js/issues/5569

And we would be able to write an example once the discussion on this is concluded:
https://github.com/ChainSafe/web3.js-plugin-chainlink/issues/1 As for now without doing something about how web3 is needed to be augmented, the example would not be clear, I think. Or what do you think? Should I write an example on the as-is for the module augmentation? Or leave it to later till we conclude a better way? 
And by the way, I am currently trying to use the plugin in a local test project to see what could be the best way to do the module augmentation. But I think that we can merge this MR independently...
Or what do you say?

Comment (mpetrunic) [reply README.md:33]: > That is right, we will add an article on how to use this plugin and it would contain examples for js and ts. This would be part of https://github.com/web3/web3.js/issues/5569

I don't think we are allowed to create documentation for this in web3.js, otherwise, we would need to allow everyone to create docs for their plugins there.

> And by the way, I am currently trying to use the plugin in a local test project to see what could be the best way to do the module augmentation. But I think that we can merge this MR independently...
Or what do you say?

I'm fine with merging PR as is if it's not easy to write example how to use it :)

Comment (Muhammad-Altabba) [reply README.md:33]: Yes, that article should not be part of web3.js repo. So it will be either as part of this repository (in a .md file for example, and this is what I think is the best). Or away from there as something on medium.com, for example.

And so I hope you approve this MR, please.
Thanks,

**Review by mpetrunic — APPROVED** [review]

File: README.md
Comment (mpetrunic) [reply README.md:33]: Oh sorry, did not know you were waiting for my approval. I'm avoiding requesting changes just so I'm not a blocker to you guys 😅

**Pull request merged**
